homeassistant/components/wyoming/assist_satellite.py

```diff
@@ -116,6 +121,10 @@ def __init__(
         self.device.set_pipeline_listener(self._pipeline_changed)
         self.device.set_audio_settings_listener(self._audio_settings_changed)
 
+        # For announcements
+        self._ffmpeg_manager = ffmpeg.get_ffmpeg_manager(hass)
+        self._played_event_received = asyncio.Event()
```

Comment:
can we initialize these when the first announcement is done? No need to always instantiate it.